Foundation/src/SharedMemory_POSIX.cpp

```diff
@@ -32,7 +32,7 @@ SharedMemoryImpl::SharedMemoryImpl(const std::string& name, std::size_t size, Sh
 	_access(mode),
 	_name("/"),
 	_fileMapped(false),
-	_server(server)
+	_server((mode == SharedMemory::AM_WRITE))
```

Comment:
if `bool server is not needed/used, it should be removed from the signature and documentation updated